scripts/ecs/wallet-production.json

```diff
@@ -0,0 +1,95 @@
+{
+    "family": "morpher-wallet-production",
+    "taskRoleArn": "arn:aws:iam::612970435624:role/morpher-wallet-backend-role",
+    "executionRoleArn": "arn:aws:iam::612970435624:role/ecsTaskExecutionRoleProduction",
+    "networkMode": "awsvpc",
+    "containerDefinitions": [
+        {
+            "name": "morpher-wallet-backend-production",
+            "image": "612970435624.dkr.ecr.eu-central-1.amazonaws.com/morpher-wallet-backend-production",
+            "cpu": 0,
+            "memory": 256,
+            "portMappings": [
+                {
+                    "containerPort": 80,
+                    "hostPort": 80,
+                    "protocol": "tcp"
+                }
+            ],
+            "essential": true,
+            "environment": [
+                {
+                    "name": "PORT",
+                    "value": "80"
+                },
+                {
+                    "name": "ENVIRONMENT",
+                    "value": "production"
+                },
+                {
+                    "name": "DB_PORT",
+                    "value": "5432"
+                },
+                {
+                    "name": "DB_DIALECT",
+                    "value": "postgres"
+                }
+            ],
+            "secrets": [
+                {
+                    "name": "DB_BACKEND_SALT",
+                    "valueFrom": "arn:aws:secretsmanager:eu-central-1:612970435624:secret:environment_dev-GRZlqk:WALLET_BACKEND_SALT::"
```

Comment:
Do we need the production secrets here?

Reply:
Yeah this branch is only for the account deletion feature. Should I merge this with the production release branch?